src/org/usfirst/frc4904/robot/commands/ArmSet.java

```diff
@@ -0,0 +1,21 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.subsystems.Arm;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ * 
+ * Once ArmMove puts the arm into
+ * the desired position, the boolean
+ * endOnArrival is true, causing
+ * the 'on' value in ArmBrakeSet to
+ * change to true, extending the
+ * piston and stopping the arm.
+ *
+ */
+public class ArmSet extends CommandGroup {
```

Comment:
Isn't this command the exact same thing as ArmMove, but with a different name? Do we need this?